tests/testthat/_snaps/conditions.md

```diff
@@ -22,24 +22,24 @@
     Output
       <error/vctrs_error_subscript_oob>
       Error in `vec_slice()`:
-      ! Can't subset elements that don't exist.
-      x Element `foo` doesn't exist.
+      ! Can't subset elements.
```

Comment:
Do we really need "Can't subset elements" in these three errors?

Reply:
The thing is that this is this header that carries the `arg` information, if any. That will be the case when called from `select()` and tibble's subset methods.